src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -2662,6 +2662,9 @@ components:
               type: integer
               format: int64
               nullable: true
+            permanently_rejects_at_epoch:
```

Comment:
1. This should result in API version bump.
2. This is a property of a specific payload, i.e. it should belong to `known_payloads` collection items.
3. The description isn't super clear to me. I'd be more verbose in this case. Say "The epoch number at witch the transaction payload is guaranteed to get permanently rejected by the Network due to exceeded epoch range/validity" or something similar? We should make it clear this is some of guideline for the end client.

Reply:
1. it's open discussion what API version is. 
2. are you sure it's not part of intent? i'm almost certain it is.
3. fixed.

Reply:
1. Yeah, there are some discussion in this area but still any API extension should result in version bump. It always did.
2. Oh, yes. My mistake!

Reply:
1. well it should result but we always matched it with release version, so in this case that would be 2.0.0